keyboards/willoucom/keypad/info.json

```diff
@@ -0,0 +1,70 @@
+{
+    "manufacturer": "Willou.com",
+    "keyboard_name": "keypad",
+    "maintainer": "willoucom",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP0", "GP1", "GP2", "GP3", "GP4"],
+            ["GP5", "GP6", "GP7", "GP8", "GP9"],
+            ["GP10", "GP11", "GP12", "GP13", "GP14"],
+            ["GP15", "GP16", "GP17", "GP18", "GP19"],
+            ["GP20"],
+            ["GP21"],
+            ["GP22", "GP26"],
+            ["GP27"],
+            ["GP28"]
```

Comment:
Each row should have the same number of columns. Please rearrange this and update your layout accordingly.

Reply:
Should I fill the rows with null values?

Reply:
No, rearrange the pins into something more rectangular.

Reply:
If the pin count is prime, then sure.